svc-api/handle/handlers.go

```diff
@@ -381,16 +380,6 @@ func GetMetadata(ctx iris.Context) {
 					models.Include{Namespace: "FabricCollection"},
 				},
 			},
-			models.Reference{URI: "http://redfish.dmtf.org/schemas/v1/FirmwareInventory_v1.xml",
-				TopInclude: []models.Include{
-					models.Include{Namespace: "FirmwareInventory"},
-				},
-			},
-			models.Reference{URI: "http://redfish.dmtf.org/schemas/v1/FirmwareInventoryCollection_v1.xml",
-				TopInclude: []models.Include{
-					models.Include{Namespace: "FirmwareInventoryCollection"},
-				},
-			},
```

Comment:
FirmwareInventory is supported in ODIMRA. this should not be removed

Reply:
I think namespace vise its referred to as SoftwareInventory